articles/azure-monitor/containers/container-insights-enable-arc-enabled-clusters.md

```diff
@@ -114,10 +114,29 @@ az k8s-extension create --name azuremonitor-containers --cluster-name <cluster-n
 
 To use [managed identity authentication](container-insights-onboard.md#authentication), add the `configuration-settings` parameter as in the following:
 
+>[!NOTE]
```

Comment:
move note to line 124, before the ARO operation

Reply:
Moved.